pkg/util/cmd.go

```diff
@@ -36,11 +36,27 @@ func CmdLogMixin(cmd *cobra.Command) *cobra.Command {
 	v := cmd.PersistentFlags().Int8P("verbose", "v", 0, "verbosity level")
 
 	setupLogger := func() {
-		ZapLogger = corezap.NewRaw(func(options *corezap.Options) {
-			level := zap.NewAtomicLevelAt(zapcore.Level(-*v))
-			options.Level = &level
-			options.Development = *dev
-		})
+		if *dev {
+			ZapLogger = corezap.NewRaw(func(options *corezap.Options) {
+				level := zap.NewAtomicLevelAt(zapcore.Level(-*v))
+				options.Level = &level
+				options.Development = *dev
+			})
+		} else {
+			// we need to create ZapLogger manually because we don't want to use Sampler, that does not support arbitrary log levels
+			encoder := zapcore.NewJSONEncoder(zap.NewProductionEncoderConfig())
+			sink := zapcore.AddSync(os.Stderr)
```

Comment:
```suggestion
			sink := zapcore.AddSync(cmd.ErrOrStderr())
```

this way it's easier to test if you want to inject error stream in the cobra command.

Reply:
thx, that was my first thought, how will I replace Stderr with my Writer